### PR TITLE
Update strings.yaml

### PR DESCRIPTION
**Pull request by mkestler-mozilla**
Adding strings for the opt-in option of crash reporting.

### CODE REVIEW

File: translations/strings.yaml [hunk 1]
```diff
@@ -422,6 +424,9 @@ crashreporter:
   dontSendButton:
     value: No thanks
     comment: User does not agree to send a crash report.
+  crashOptInDialogDescription:
+    value:  Don't show this again.  Auto-send crash reports to Mozilla.
```
Comment (lesleyjanenorton) [inline translations/strings.yaml:428]: *Don’t (apostrophe)

File: translations/strings.yaml [hunk 2]
```diff
@@ -422,6 +424,9 @@ crashreporter:
...
+    comment:  This is a descrption for a checked option on the crash reporter dialog.
```
Comment (lesleyjanenorton) [inline translations/strings.yaml:429]: *description

File: translations/strings.yaml (hunk 1 above)
Comment (flodolo) [inline translations/strings.yaml:428]:
```suggestion
    value:  Don't show this again. Auto-send crash reports to Mozilla.
```

Drop the double space after punctuation.

Does `Auto-send` come from our Content Team? I've never seen it before, it's normally spelled out.

**Review by flodolo — CHANGES_REQUESTED** [review]

File: translations/strings.yaml (hunk 2 above)
Comment (flodolo) [inline translations/strings.yaml:429]:
```suggestion
    comment:  This is a description for a checkbox on the crash reporter dialog.
```

File: translations/strings.yaml (hunk 1 above)
Comment (mkestler-mozilla) [reply translations/strings.yaml]: Auto-send was in the figma, but I've changed it.  I think Autosend as one word is the correct usage.

File: translations/strings.yaml (hunk 2 above)
Comment (flodolo) [reply translations/strings.yaml:429]: This is still not fixed.

Comment (mkestler-mozilla) [reply translations/strings.yaml]: Forgot to push.  It is fixed now.

**Review by flodolo — APPROVED** [review]

**Pull request merged**